### DISCUSSION TITLE
Add Private option and Source field to torrent creation dialog

### DISCUSSION THREAD

**Discussion by rflois** [openscopeproject/TrguiNG#59] [Ideas]
For many private trackers, these fields are necessary. Can these be added as options? It would definitely speed up my process to be able to run the hashing directly on my server. Thanks!

**Comment by qu1ck** [comment]
Source and private fields added in [1c0d926](https://github.com/openscopeproject/TrguiNG/commit/1c0d92603d426fa927139fe603e95a1bb8426cd8)

I'm not sure if I understood you but path mappings are irrelevant for torrent creation, that is entirely local process and has nothing to do with the server.

**Reply by rflois** [reply]
I mentioned the path mappings in the app crash for torrent creation, in case path mappings had anything to do with it :). Thanks so much for this!

**Reply by rflois** [reply]
And confirmed this works as well. Thanks!
